projects/packages/assets/src/class-assets.php

```diff
@@ -458,4 +485,196 @@ public static function wp_default_scripts_hook( $wp_scripts ) {
 		$wp_scripts->add_inline_script( 'wp-jp-i18n-state', $js, 'before' );
 	}
 
+	// endregion .
+
+	// ////////////////////
+	// region Textdomain aliasing
```

Comment:
Would it make sense to move all those new methods to a new class, since they're all focused on a specific part of the process (i18n)?

Reply:
We could. On the other hand, having `Assets::` as the static entry point is kind of nice.